bigtable-dataflow-parent/bigtable-dataflow-import/src/main/java/com/google/cloud/bigtable/dataflowimport/HadoopFileSource.java

```diff
@@ -102,6 +105,9 @@
  */
 public class HadoopFileSource<K, V> extends BoundedSource<KV<K, V>> {
   private static final long serialVersionUID = 0L;
+  private static final Logger SOURCE_LOG = LoggerFactory.getLogger(HadoopFileSource.class);
+
+  private static final long ONE_HUNDRED_MB = 100 * (1 << 20);
```

Comment:
Name it by how it is used, not the value is has. Perhaps MIN_BUNDLE_SIZE_BYTES? or MIN_BUNDLE.